[2022-03-26 06:00:01,944] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-03-26T00:00:00+00:00 [queued]>
[2022-03-26 06:00:01,951] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-03-26T00:00:00+00:00 [queued]>
[2022-03-26 06:00:01,951] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-26 06:00:01,951] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-03-26 06:00:01,951] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-26 06:00:01,961] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): enter_virtual_env> on 2022-03-26T00:00:00+00:00
[2022-03-26 06:00:01,965] {standard_task_runner.py:52} INFO - Started process 507626 to run task
[2022-03-26 06:00:01,970] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matchup', 'enter_virtual_env', '2022-03-26T00:00:00+00:00', '--job-id', '486', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matchup.py', '--cfg-path', '/tmp/tmpzappk1p3', '--error-file', '/tmp/tmp8i_nbg8t']
[2022-03-26 06:00:01,971] {standard_task_runner.py:77} INFO - Job 486: Subtask enter_virtual_env
[2022-03-26 06:00:02,006] {logging_mixin.py:104} INFO - Running <TaskInstance: matchup.enter_virtual_env 2022-03-26T00:00:00+00:00 [running]> on host 531edfb863fc
[2022-03-26 06:00:02,032] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=r.chesne@gmail.com
AIRFLOW_CTX_DAG_OWNER=Robin
AIRFLOW_CTX_DAG_ID=matchup
AIRFLOW_CTX_TASK_ID=enter_virtual_env
AIRFLOW_CTX_EXECUTION_DATE=2022-03-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-26T00:00:00+00:00
[2022-03-26 06:00:02,033] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-26 06:00:02,034] {bash.py:158} INFO - Running command: . /app/env/bin/activate
[2022-03-26 06:00:02,045] {bash.py:169} INFO - Output:
[2022-03-26 06:00:02,051] {bash.py:177} INFO - Command exited with return code 0
[2022-03-26 06:00:02,070] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=matchup, task_id=enter_virtual_env, execution_date=20220326T000000, start_date=20220326T060001, end_date=20220326T060002
[2022-03-26 06:00:02,094] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-26 06:00:02,101] {local_task_job.py:146} INFO - Task exited with return code 0
